packages/devtools_app/lib/src/screens/profiler/panes/method_table/method_table.dart

```diff
@@ -38,6 +39,8 @@ class CpuMethodTable extends StatelessWidget {
   }
 }
 
+// TODO(kenz): ensure that this table automatically scrolls to the selected
```

Comment:
When would this happen?

Reply:
If you select a method from anywhere in the method graph (callees and callers), the graph reroots to that method, which means the method table should scroll to the new selection. We may eventually want to maintain some sort of history here for this reason.